site-modules/gplt_runner_setup/manifests/user_setup.pp

```diff
@@ -51,6 +57,12 @@
     group  => $test_user,
   }
 
+  file { "${test_user} html users link":
+    ensure => "link",
```

Comment:
I have had issues with nginx being able to traverse linked directories.  Has this been tested?  Also, should we just make this the default `www_root` directory in nginx and not not link it?

Reply:
I have as well, took some work to figure it out.  selinux was the main thing breaking it.  Which is why selinux is turned down as part of the instance setup.  I did test this on a new setup and it worked as expected.   And as for making it the default... see the comment above about multiple users.